Ginger/Ginger/Actions/ActionEditPages/Communication/ActeMailEditPage.xaml.cs

```diff
@@ -229,11 +229,10 @@ private void BindSendEMailConfigView()
             xEmailConfigView.HasAttachmentsSelectionChanged += xSendEMailConfigView_HasAttachmentsSelectionChanged;
             xEmailConfigView.ReadmailMethodChanged += xReadEmailConfigView_ReadMethodChanged;     
             xEmailConfigView.AddFileAttachment += xSendEMailConfigView_FileAdded;
+            xEmailConfigView.AttachmentNameVEButtonClick += xSendEMailConfigView_NameValueExpressionButtonClick;
```

Comment:
that was decided with Mahesh K. to remove the Weak Binding and revert to direct binding?